pymeasure/instruments/hp/hpsystempsu.py

```diff
@@ -0,0 +1,330 @@
+
+    def clear(self):
+        """
+        Resets the instrument to power-on default settings
+
+        """
+        self.write("CLR")
+
+    delay = HPLegacyInstrument.setting(
+        "DELAY %g",
+        """
+        A float propery that changes the reprogamming delay
+        Default values:
+        8 ms in FAST mode
+        80 ms in NORM mode
+
+        Values will be rounded to the next 4 ms by the instrument
+
+        """,
+        validator=strict_range,
+        values=[0, 32.768],
+    )
+
+    display_active = HPLegacyInstrument.setting(
+        "DIS %d",
+        """
+        A boot property which controls if the display is enabled
+        """,
+        validator=strict_discrete_set,
+        values={False: 0, True: 1},
+        map_values=True
+    )
+
+    @property
+    def status(self):
+        """
+        Returns an object representing the current status of the unit.
+
+        """
+        # overloading the already exisiting propery because of the different command
+        reply = bytearray(int(self.adapter.connection.query("STS?")).to_bytes(
+            self.status_bytes_count, "little"))
+        return self.status_bits.from_buffer(reply)
+
+    id = HPLegacyInstrument.measurement(
+        "ID?",
+        """
+        Reads the ID of the instrument and returns this value for now
+
+        """,
+    )
+
+    current = HPLegacyInstrument.control(
+        "IOUT?", "ISET %g",
+        """
+        A floating point property that controls the output current of the device.
+
+        """,
+        validator=strict_range,
+        values=[0, limits["HP6632A"]["Cur_lim"]],
+    )
+
+    over_voltage_limit = HPLegacyInstrument.setting(
+        "OVSET %g",
+        """
+        A floationg point property that sets the OVP threshold.
+
+        """,
+        validator=strict_range,
+        values=[0, limits["HP6632A"]["OVP_lim"]],
+    )
+
+    OCP_enabled = HPLegacyInstrument.setting(
+        "OCP %d",
+        """
+        A bool property which controls if the OCP (OverCurrent Protection) is enabled
+        """,
+        validator=strict_discrete_set,
+        values={False: 0, True: 1},
+        map_values=True
+    )
+
+    output_enabled = HPLegacyInstrument.setting(
+        "OUT %d",
+        """
+        A bool property which controls if the outputis enabled
+        """,
+        validator=strict_discrete_set,
+        values={False: 0, True: 1},
+        map_values=True
+    )
+
+    @output_enabled.getter
+    def output_enabled(self):
+        """
+        A bool property which controls if the output is enabled
+        """
+        output_status = bool(self.status.CV or self.status.CCpos or
+                             self.status.CCneg or self.status.Unregulated)
+        return output_status
+
+    def reset_OVP_OCP(self):
+        """
+        Resets Overvoltage and Overcurrent protections
+
+        """
+        self.write("RST")
+
+    rom_Version = HPLegacyInstrument.measurement(
```

Comment:
I am not a big fan (and expert) of code style, but sometime improves code readability.
`pymeasure` follows https://www.python.org/dev/peps/pep-0008/

See also https://peps.python.org/pep-0008/#function-and-variable-names

In this case, I think it would be better to use `rom_version`.

Could You please check also other names in this module (take into account that exceptions are allowed when necessary) ?

Reply:
agreed, will change to `rom_version`